examples/fan-in/main.pony

```diff
@@ -277,5 +277,6 @@ class Tick is TimerNotify
     fun ref apply(timer: Timer, count: U64): Bool =>
       _tick_count = _tick_count + count
       let done = (_report_count > 0) and (_tick_count >= _report_count)
+      @printf(("Tick fired: " + _tick_count.string() + "\n").cstring())
```

Comment:
did you intend to have this in here or was it purely from testing? this seems like testing. in an example if this was meant to stay, i wouldnt want to use ffi for printing.

Reply:
yep.. removed.